src/wp-includes/formatting.php

```diff
@@ -4696,7 +4696,7 @@ function esc_attr( $text ) {
 	 *
 	 * @since 2.0.6
 	 *
-	 * @param string $safe_text The text after it has been escaped.
+	 * @param string $safe_text The text after it has been sanitized and escaped.
```

Comment:
`esc_attr()` is an escaping function, as WordPress uses the term sanitization for user input this could be confusing to developers.

```suggestion
	 * @param string $safe_text The text after it has been escaped.
```

Reply:
HI @peterwilsoncc,
Should I open a follow-up PR to address this issue?

Reply:
@Infinite-Null Don't worry, it was more a nit pick than a particular concern.

Sorry about the coincidence of timing as I commented and Sergey committed. These things happen occasionally :)

Reply:
No worries at all! I appreciate the clarification. Thank You 😄